config/deploy/production.rb

```diff
@@ -15,6 +15,9 @@
   before :finished, 'newrelic:notice_deployment'
 end
 
+set :slack_webhook, 'https://hooks.slack.com/services/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'
```

Comment:
This repo is public, so we need to revoke this key and generate a new one and configure this to pull the key from ENV